react/src/react-table-config.d.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * This config file is required for hooks to work in Table.tsx.
```

Comment:
I wasn't able to use some built-in hooks in react-table and followed the steps here: https://github.com/tannerlinsley/react-table/blob/master/TYPESCRIPT.md to fix it